code-challenges.test.js

```diff
@@ -10,42 +11,197 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
+
 // --------------------1) Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized.
 
 // a) Create a test with an expect statement using the variable provided.
 
-const people = [
+
+// a describe method that lists the name of the function OR naming of the particular test.
+describe("jobs", () => {
+
+  // a test/it method, nested within the describe block, that in plain words, describes that the function does.
+  it(" that takes in an array of objects and returns an array with a sentence about each person with their name capitalized.", () => {
+
+
+ const people = [
   { name: "ford prefect", occupation: "a hitchhiker" },
   { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
   { name: "arthur dent", occupation: "a radio employee" }
 ]
-// Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
 
+    //an expect method, nested within the test block, calling on the hello() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+    expect(jobs(people)).toEqual(["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."])
+  })
+})
```

Comment:
Beautiful test!

Reply:
thank you i tried to keep my names simple picking them ahead of time. Placing my tests so i could keep my code cleaner.